packages/blockchain-link-utils/src/blockbook.ts

```diff
@@ -1,3 +1,7 @@
+// @ts-expect-error
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
please add the lib to `package.json` instead of disabling errors 😢

Reply:
my bad, https://github.com/trezor/trezor-suite/pull/13267/commits/c3586f7e269c64a49eb2481a5b0aedd9af653c5c